src/templates/en/2019/contributors.html

```diff
@@ -3,3 +3,16 @@
 {% block title %}
 Contributors | The Web Almanac by HTTP Archive
 {% endblock %}
+
+{% block main %}
+{% for contributor in contributors %}
+<section>
+  <h3>{{ contributor.name }}</h3>
+  <ul>
+    <li>{{ team.name }}</li>
```

Comment:
We'll need another loop to render an `<li>` for each team that the contributor is a part of.
```suggestion
    {% for team in contributor.teams %}
    <li>{{ teams[team].name }}</li>
    {% endfor %}
```